.github/scripts/radius-bot.js

```diff
@@ -129,6 +132,60 @@ async function cmdOkToTest(github, issue, isFromPulls, username) {
     }
 }
 
+
+
+/**
+ * Trigger update bicep for the pull request.
+ * @param {*} github GitHub object reference
+ * @param {*} issue GitHub issue object
+ * @param {boolean} isFromPulls is the workflow triggered by a pull request?
+ * @param {string} username is the user who trigger the command
+ */
+async function cmdUpdateBicep(github, issue, isFromPulls, username) {
+    if (!isFromPulls) {
+        console.log('[cmdUpdateBicep] only pull requests supported, skipping command execution.');
+        return;
+    }
+
+    // Check if the user has permission to trigger update bicep with an issue comment
+    const org = 'radius-project';
+    console.log(`Checking team membership for: ${username}`);
+    const isMember = await checkTeamMembership(github, org, process.env.TEAM_SLUG, username);
+    if (!isMember) {
+        console.log(`${username} is not a member of the ${teamSlug} team.`);
+        return;
+    }
+
+    // Get pull request
+    const pull = await github.rest.pulls.get({
+        owner: issue.owner,
+        repo: issue.repo,
+        pull_number: issue.number,
+    });
+
+    if (pull && pull.data) {
+        // Get commit id and repo from pull head
+        const testPayload = {
+            pull_head_ref: pull.data.head.sha,
+            pull_head_repo: pull.data.head.repo.full_name,
+            command: 'update-bicep',
+            issue: issue,
+        };
```

Comment:
nit: I don't think `testPayload` fits in this context. Maybe call it `requestPayload` or just `payload`?

```suggestion
        const requestPayload = {
            pull_head_ref: pull.data.head.sha,
            pull_head_repo: pull.data.head.repo.full_name,
            command: 'update-bicep',
            issue: issue,
        };
```